src/style.css

```diff
@@ -0,0 +1,47 @@
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.title {
+  display: none;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: -2;
```

Comment:
a couple of issues here:
- paddings should be used alongside certain measurement unit (`px`, `em`, `rem`, etc.)
- this rule is hardcoded, it is not supposed to be here, let's find another solution